.gitignore

```diff
@@ -126,3 +126,4 @@ Migrations/
 /test-old-libraries/target/
 /e2e-tests/spring-web/target/
 /e2e-tests/spring-rest-mongo/target/
+/core/src/main/resources/openapi_pet.json
```

Comment:
why this?

Reply:
Hi Andrea, 

Thank you for your comment. I used that file in the same directory as the main executable just to regenerate bug #701 before fixing and running test cases. I have removed that line since it is not necessary.